apps/prairielearn/elements/pl-dataframe/pl-dataframe.py

```diff
@@ -142,7 +142,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         descriptors = frame.agg([get_dtype_function]).set_axis(
             ["dtype"], axis="index", copy=False
         )
-        other = descriptors.style.applymap(lambda v: "font-weight: bold;")
+        other = descriptors.style.map(lambda v: "font-weight: bold;")
```

Comment:
This is actually correct, but getting typecheck errors because pandas stubs haven't updated yet.